eventuals/static-thread-pool.cc

```diff
@@ -167,6 +165,13 @@ bool StaticThreadPool::Continuable(Context* context) {
 
 ////////////////////////////////////////////////////////////////////////
 
+void StaticThreadPool::Clone(Context& child, Context& parent) {
+  // Storing additional requirements pinned core.
+  child.data_ = Scheduler::Context::Get()->data_;
```

Comment:
I understand that this is the same thing but:
```suggestion
  child->data = parent->data;
```
Thinking more about this, why don't we drop the `parent` argument to `Clone()` altogether and instead we can do:
```suggestion
  child->data = child->parent()->data;
```

And please leave a comment why it's ok to reuse the parent's data pointer without allocating a new one.

Reply:
Ok, final comment here, let's remove the `parent` argument and then we can keep the code just as you had it, but still add the comment!